api/app/clients/chatgpt-client.js

```diff
@@ -7,6 +7,7 @@ const askClient = async ({
   parentMessageId,
   conversationId,
   model,
+  token,
```

Comment:
if this is for setting custom api key, we shouldn't refer to it as token, especially since we use the term "token" in a totally different context. That being the case, openAiApiKey would be a better name.